src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 summary: Wizard splits complex tasks and process into steps.
 ---
 
-# Wizard
+# Wizard UI Pattern
+You can use the Wizard UI pattern to split large complex tasks and processes into smaller manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
 
-Split complex tasks and process into steps.
+**How to Use the Wizard UI Pattern**
 
-Use the Wizard to present numbered steps or conditions that the end user needs to complete or reach a goal. Use it to split large tasks into more manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
+1. In Service Studio, in the Toolbox, search for  `Wizard`. 
 
-**How to use**
+    The Wizard widget is displayed.
```

Comment:
The Wizard widget and the Wizard item are displayed. You'll need both to use this Pattern.

Reply:
Edited: The Wizard and Wizard Item widgets are displayed (both of which are required to use this pattern).